client/gutenboarding/style.scss

```diff
@@ -1,11 +1,62 @@
 @import './gutenberg-styles/styles';
 
+// Override core variables: we don't have wp-admin header and sidebar in Calypso
+$admin-bar-height: 0;
+$admin-bar-height-big: 0;
+$admin-sidebar-width: 0;
+$admin-sidebar-width-big: 0;
+$admin-sidebar-width-collapsed: 0;
+
+// Calypso section wrapper resets
+.is-section-gutenboarding {
+	background: $white;
+
+	.layout__content {
+		margin: 0;
+		padding: 0;
+	}
+}
+
 .gutenboarding {
+	// Reset some Calypso base styles
+	input[type='text'],
+	input[type='search'],
+	input[type='email'],
+	input[type='number'],
+	input[type='password'],
+	input[type='checkbox'],
+	input[type='radio'],
+	input[type='tel'],
+	input[type='url'],
+	textarea {
+		width: auto;
+	}
+
+	// Gutenberg reset
 	@include reset;
+
+	// Gutenberg styles
+	@import '~@wordpress/edit-post/src/style.scss';
+	@import '~@wordpress/components/src/style.scss';
+	@import '~@wordpress/block-editor/src/style.scss';
+	@import '~@wordpress/format-library/src/style.scss';
 }
 
 .gutenboarding__block-editor *,
 .gutenboarding__block-editor ::after,
 .gutenboarding__block-editor ::before {
 	box-sizing: border-box;
 }
+
+// These are default block editor styles in case the theme doesn't provide them.
+.wp-block {
```

Comment:
If we'll have only onboarding block in the editor without alignment options, maybe we don't need this?